demo/tmail/mailetcontainer.xml

```diff
@@ -98,9 +98,10 @@
             </mailet>
 
             <!-- Put the OpenAIMailet after LocalDelivery so the GPT reply would come after the asking question -->
-            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.OpenAIMailet">
+            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.MailBotMailet">
```

Comment:
```suggestion
            <mailet match="com.linagora.tmail.mailet.RecipientsContain=gpt@tmail.com" class="com.linagora.tmail.mailet.AIBotMailet">
```

?

Reply:
Okay no problem, but I would have to rename it everywhere in a separate commit.